coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/loader/sql/SqlDelogger.scala

```diff
@@ -183,6 +183,8 @@ class SqlDelogger[CV](connection: Connection,
           null
         case SqlLogger.SecondaryReindex =>
           Delogger.SecondaryReindex
+        case SqlLogger.SecondaryAddIndex =>
+          Delogger.SecondaryReindex
```

Comment:
This should be `secondaryAddIndex(aux)`, where that looks like:

```scala
def secondaryAddIndex(aux: Array[Byte]) = {
  val msg = messages.SecondaryAddIndex.defaultIndex.mergeFrom(aux)
  Delogger.SecondaryAddindex(ColumnName(msg.fieldName))
}
```

(+/- typos because I just entered that off the top of my head; there are lots of examples of this below)